microsoft-edge/webview2/release-notes.md

```diff
@@ -34,6 +34,149 @@ To load WebView2, the minimum version of Microsoft Edge or the WebView2 Runtime
 
 To use a prerelease SDK along with a Microsoft Edge preview channel, see [Test upcoming APIs and features](how-to/set-preview-channel.md).
 
+<!-- ====================================================================== -->
+## 1.0.1462.37
+
+Release Date: December 12, 2022
+
+[NuGet package for WebView2 SDK 1.0.1418.22](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1462.37)
+
+For full API compatibility, this version of the WebView2 SDK requires WebView2 Runtime version 108.0.1462.37 or higher.
+
+### General
+
+This WebView2 SDK release has the same bug fixes that are in WebView2 SDK 1.0.1549-prerelease. See **Bug fixes** in the following section.
+
+
+<!-- ====================================================================== -->
+## 1.0.1549-prerelease
+
+Release Date: December 12, 2022
+
+[NuGet package for WebView2 SDK 1.0.1466-prerelease](https://www.nuget.org/packages/Microsoft.Web.WebView2/1.0.1549-prerelease)
+
+For full API compatibility, this version of the WebView2 SDK requires Microsoft Edge version 110.0.1549.0 or higher.
+
+### General
+
+#### Experimental features 
+
+*  Added support for the Locale Region API:
+
+##### [.NET/C#](#tab/dotnetcsharp)
+

```

Comment:
near
https://learn.microsoft.com/en-us/dotnet/api/microsoft.web.webview2.core.corewebview2environmentoptions.iscustomcrashreportingenabled?view=webview2-dotnet-1.0.1466-prerelease#microsoft-web-webview2-core-corewebview2environmentoptions-iscustomcrashreportingenabled
but change the version number.

Reply:
fixed